### PR TITLE
[WIP] Update deployment instructions for 4.0 milestone 4

### LINKED ISSUE #205: Describe installing CaaSP from SCC
We have the SCC setup now, meaning CaaSP is installable now as an extension.

The product is in the Alpha stage, meaning only internal SUSE employees can access to it. So we need to document 2 different things:

1- How to use SCC as an employee for the Alpha (release notes or announcement email)
2- How to use SCC in general (deployment)

Later on, we will have to adapt 1 to Beta users.

# How to use SCC as an employee for the Alpha

1- Go to https://scc.suse.com/admin/products/1863 and click on "I want a reg code!"
2- Follow the steps. Click on "Activate ..." Warning: Do not select Novell Inc as an organization for performance usage.
3- Now you should have a subscription code in https://scc.suse.com/admin/products/1863

Use this code in "How to use SCC in general"

# How to use SCC in general

You need a registration code. With that, you can use "SUSEConnect -p caasp/4.0/x86_64 -r REGCODE " and then install the packages .... (more info on what to install?)

# How to use the beta codes

We will have a list of beta codes. PM should share them with the customers? How a customer requests a code? in the mailing list???

### PR DESCRIPTION
**Pull request by r0ckarong**
Closes #205

### CODE REVIEW

**Comment by r0ckarong** [comment]
[caasp-deployment_color_en.pdf](https://github.com/SUSE/doc-caasp/files/3161970/caasp-deployment_color_en.pdf)

File: adoc/deployment-sles.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+If you already have a running {sle} 15 SP1 installation, you can add {productname}
+to this installation using SUSE Connect. You also need to enable the "Server Applications"
+and "Containers" modules because they contains some dependencies required by
+{productname}.
+
+Retrieve your {productname} registration code and run:
+
+----
+SUSEConnect -p sle-module-containers/15.1/x86_64
+SUSEConnect -p sle-module-server-applications/15.1/x86_64
```
Comment (kravciak) [inline adoc/deployment-sles.adoc]: do we need server-applications?

Comment (r0ckarong) [reply adoc/deployment-sles.adoc]: @jordimassaguerpla There was a package that was moved from server applications to base system right? If so let's remove this requirement.

Comment (jordimassaguerpla) [reply adoc/deployment-sles.adoc]: exactly, socat. But now server applications is not needed anymore.

**Comment by r0ckarong** [comment]
Relates to: https://github.com/SUSE/caaspctl/pull/193

**Pull request merged**